cafebabel/templates/articles/macros.html

```diff
@@ -8,6 +8,14 @@
   {% endif %}
 {% endmacro %}
 
+{% macro profile_image(author) %}
+  {% if author.profile.has_image %}
+    <div style="background-image:url({{ author.profile.image_url }})" class=image></div>
+  {% else %}
+    {{ default_profile_picture() }}
```

Comment:
the macro `default_profile_picture` is now only used here.
Therefore it should be deleted and it's code moved here.